resources/views/global/_latest-news-slider.antlers.html

```diff
@@ -0,0 +1,32 @@
+
+<div class="global-news-slider relative py-90 bg-concrete">
```

Comment:
Our naming structure ould be something like this...
```suggestion
<div class="global-newsSlider relative py-90 bg-concrete">
```